app/Providers/AppServiceProvider.php

```diff
@@ -14,7 +17,13 @@ class AppServiceProvider extends ServiceProvider
      */
     public function register()
     {
-        //
+        $this->app->bind(RequestActivityLoggerInterface::class, function () {
+            if (App::environment('production')) {
+                return $this->app->make(RequestActivityLoggerInterface::class);
+            } else {
+                return $this->app->make(DummyRequestActivityLogger::class);
```

Comment:
DebugRequestActivityLogger was requested to be injected.

Reply:
my mistake i chanege this